app/assets/stylesheets/views/_fatality-notice.scss

```diff
@@ -1,3 +1,5 @@
 .fatality-notice {
-  @include sidebar-with-body;
+  .app-c-publisher-metadata {
```

Comment:
Overriding component CSS isn't ideal, but not sure at the moment how we'd do it another way.

Reply:
I'd expect this to be updated/removed once we've had a conversation with MoD to understand the needs for this format, we intend to provide sidebar links, but we need to build some additional functionality before adding them to this format, so this should be a temporary update.